config/eslint/eslintrc.js

```diff
@@ -174,7 +174,7 @@ module.exports = {
       },
     ],
     "@typescript-eslint/unified-signatures": "error",
-    "@typescript-eslint/return-await": "error",
+    "@typescript-eslint/return-await": "error", // keeping this line only
```

Comment:
This comment is unnecessary, please remove.

```suggestion
    "@typescript-eslint/return-await": "error",
```